.circleci/config.yml

```diff
@@ -24,9 +24,25 @@ jobs:
     docker:
       - image: circleci/golang:<< parameters.go_version >>
         user: root
+      - image: postgres:14
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: postgres
+          POSTGRES_PASSWORD: mysecretpassword
     working_directory: /go/src/github.com/instana/go-sensor
     steps:
       - checkout
+      - run:
+          name: Waiting for Postgres to be ready
+          command: |
+            for i in `seq 1 10`;
+            do
+              nc -z localhost 5432 && echo Success && exit 0
```

Comment:
[Optional] Maybe we could add the env var `PGPORT` after line 31 to match the port 5432 from the "waiting" script. On the other hand, 5432 is the Postgres default port, so not adding it is also fine.